src/test/java/htmlverification/service/CvsCertificateTestDataProvider.java

```diff
@@ -149,6 +157,7 @@ public static VTP30 getVtp30() {
                 .setImageType("png");
 
         vtp30.setSignature(signature);
+        vtp30.setReissue(generateReissuer());
```

Comment:
Do we not want to use replacement as terminology for reissue, just thinking of consistency across repos as it might be a bit confusing otherwise.

Reply:
Or is the idea for a reissue that there can be multiple reasons which includes replacements?

Reply:
There are some other reasons for a reissue that we need to handle at a later date. These changes are just for replacements.

Reply:
replacement and duplicate are the 2 scenarios we know about so I went more generic as both are ReIssues of some kind